src/ebonite/build/runner/docker.py

```diff
@@ -16,7 +16,9 @@ def instance_type(self) -> Type[DockerContainer]:
         return DockerContainer
 
     def create_instance(self, name: str, ports_mapping: Dict[int, int] = None, **kwargs) -> DockerContainer:
-        return DockerContainer(name, ports_mapping)
+        if 'ports_mapping' in kwargs:
```

Comment:
it's better to do something like `ports_mapping = kwargs.pop('ports_mapping', ports_mapping)`. that way you will not loose parameter if it was in kwargs and also eliminate redundant if